src/components/Code/Code.js

```diff
@@ -41,8 +41,57 @@ const Pre = styled.pre`
   font-size: ${props => props.theme.fontSizes.small};
 `;
 
+const PaddedLine = styled.div`
+  &:not(:last-child) {
+    padding-bottom: 8px;
+  }
+
+  /* required to make jsx children work with adding '\n' in multiLine */
+  & > div {
```

Comment:
This assumes something about the DOM structure of children passed to `<Code />` component, so I'd rather avoid it if possible. Would a `& > *` selector do the trick?

Reply:
Actually wouldn't this already work if just `{'\n'}` was put outside of `<PaddedLine>` block below?

Reply:
Would require another wrapper for adjacent jsx elements as we don't have `React.Fragment` yet. Have changed to `*` selector

Reply:
Ok - I think having another wrapper wouldn't be such a big deal either (in case this proves to be non-ideal) :)